Changelog.md

```diff
@@ -62,6 +62,7 @@ Breaking Changes:
  * Type Checker: Disallow "loose assembly" syntax entirely. This means that jump labels, jumps and non-functional instructions cannot be used anymore.
  * Type System: Disallow explicit and implicit conversions from decimal literals to ``bytesXX`` types.
  * Type System: Disallow explicit and implicit conversions from hex literals to ``bytesXX`` types of different size.
+ * Type System: IntegerType is split into IntegerType and AddressType internally.
```

Comment:
Is this really a breaking change? Perhaps "Compiler Feature" would be better suited.

Reply:
It is in the sense that ``address x = true ? 0 : address(this);`` was valid before - in that sense maybe a bugfix, though...

Reply:
But then you should probably add a second entry just explaining this one.